src/Containers/SessionChoice/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { Col, Row } from 'antd';
+import React, { useState } from 'react';
+
+import Button from 'Components/Button';
+import Input from 'Components/Input';
+import { Box, CenteredContainer, OptionalText, Tags } from './styles';
+import pushNotification from 'pushNotification';
+
+interface Payload {
+  nickname: string;
+  game_code: string;
+}
+
+interface Response {
+  player_id: number;
+  game_code: number;
+  token: string;
+}
+
+const SessionChoice = () => {
+  const [nickname, setNickname] = useState<string>('');
+  const [gameCode, setGameCode] = useState<string>('');
+
+  const newSessionButton = 'Create new session';
+  const joinSessionButton = 'Join session';
+
+  const onSubmit = async () => {
+    const info: Payload = { nickname, game_code: gameCode };
+    const response = await fetch('http://127.0.0.1:8000/api/v1/game/', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(info),
+    });
+    if (response.ok) {
+      const result: Response = await response.json();
+      localStorage.setItem('token', result.token);
+      localStorage.setItem('player_id', String(result.player_id));
+      localStorage.setItem('game_code', String(result.game_code));
+      pushNotification('success', 'Joining server', 'Enjoy game', 3);
+    } else {
+      pushNotification('warning', 'Server not found', 'Please check game code', 3);
```

Comment:
We have standardized error messages
![image](https://github.com/MendelDamian/plemiona-web/assets/44109041/4a6262f3-9130-42ea-bf4f-77d70386c184)
You can make use of that and display title and description of warning/error using that

Reply:
Also, 3 seconds may be too short for user to fully get familiar with the notification